lib/actions/appleMusic.js

```diff
@@ -145,8 +153,12 @@ function appleMusic(player, values) {
                 }, []);
 
                 return player.coordinator.clearQueue()
-                  .then(() => player.coordinator.setAVTransport(queueURI, ''))
                   .then(() => player.coordinator.addURIToQueue(queueTracks[0].uri, queueTracks[0].metadata, true, 1))
+                  .then(() => {
+                      if (radioOn(player)) {
+                        player.coordinator.setAVTransport(queueURI,'');
```

Comment:
You forgot to return the promise here, it won't wait for this call to finish unless you return the promise